src/test/java/com/serenitydojo/fruitmarket/TheCatalog.java

```diff
@@ -10,49 +11,43 @@
 
 public class TheCatalog {
 
-    @DisplayName("The catalog should return the current price of a fruit")
+    Catalog catalog = new Catalog();
+
+    @BeforeEach
+    public void setUpCatalog(){
+        catalog.setPrice("Apples", 4.00);
+        catalog.setPrice("Oranges", 5.50);
+        catalog.setPrice("Bananas", 6.00);
+        catalog.setPrice("Pears", 4.50);
+    }
+    @DisplayName("Should return the current price of a fruit")
         @Test
         public void shouldBeAbleToUpdateTheCurrentPriceOfAFruit() {
-            Catalog catalog = new Catalog();
-            catalog.setPrice("Apples", 4.00);
-            catalog.setPrice("Oranges", 5.50);
-            catalog.setPrice("Bananas", 6.00);
-            catalog.setPrice("Pears", 4.50);
-
             double priceOfApples = catalog.getPriceOf("Apples");
             double priceOfOranges = catalog.getPriceOf("Oranges");
             double priceOfBananas = catalog.getPriceOf("Bananas");
-            double priceOfPears = catalog.getPriceOf("Apples");
+            double priceOfPears = catalog.getPriceOf("Pears");
 
-
-            assertThat(priceOfApples).isEqualTo(4.00);
-            assertThat(priceOfApples).isEqualTo(4.00);
             assertThat(priceOfApples).isEqualTo(4.00);
-            assertThat(priceOfApples).isEqualTo(4.00);
-
+            assertThat(priceOfOranges).isEqualTo(5.50);
+            assertThat(priceOfBananas).isEqualTo(6.00);
+            assertThat(priceOfPears).isEqualTo(4.50);
         }
 
+    @DisplayName("Should list the names of the currently available fruits in alphabetical order")
+
         @Test
         public void shouldListProductsInAlphabeticalOrder(){
-            Catalog catalog = new Catalog();
-            catalog.setPrice("Cherries", 3.00);
-            catalog.setPrice("Apples", 4.00);
-            catalog.setPrice("Bananas", 2.00);
-
-            assertThat(catalog.getProducts()).containsExactly("Apples", "Bananas", "Cherries");
+            assertThat(catalog.getProducts()).containsExactly("Apples", "Bananas", "Oranges", "Pears");
         }
 
 
+    @DisplayName("Should throw a FruitUnavailableException if the fruit is not currently available")
 
         //@Test(expected = FruitUnAvailableException.class)
         @Test
```

Comment:
```suggestion
        @Test(expected = FruitUnAvailableException.class)

```

we should test that exception gets thrown, right?